.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,21 +1,27 @@
 ---
 name: Bug
-about: "Report confirmed bugs. For unconfirmed bugs, please visit our mailing list https://groups.google.com/forum/#!forum/janusgraph-users"
+about: "Report confirmed bugs. For unconfirmed bugs, please visit our mailing list https://lists.lfaidata.foundation/g/janusgraph-users/topics"
 labels: "kind/bug/possible"
 
 ---
-
-Please post all questions and issues on [janusgraph-users](https://groups.google.com/forum/#!forum/janusgraph-users)
+<!--
+Please post all questions and issues on [janusgraph-users](https://lists.lfaidata.foundation/g/janusgraph-users/topics)
 before opening a GitHub issue. Your questions will reach a wider audience there,
-and if we confirm that there is a bug, then you can open a new issue.
+and if we confirm that there is a bug, then you can open a new issue. You could also use [GitHub Discussions](https://github.com/JanusGraph/janusgraph/discussions) or [Gitter](https://gitter.im/janusgraph/janusgraph).
+
+Reminder check your current version and please update to a supported version of JanusGraph, 
+see https://docs.janusgraph.org/changelog/#currently-supported.
+-->
 
 Please include configurations and logs if available.
 
 For confirmed bugs, please report:
-- Version: 
-- Storage Backend: 
-- Mixed Index Backend:
-- Mailing list Thread URL:
+- Version: <!-- e.g.: 0.5.0,0.6.0 -->
+- Storage Backend: <!-- e.g.: cql, hbase, inmemory -->
+- Mixed Index Backend: <!-- e.g.: elasticsearch, none -->
+- Mailing list Thread or Gitter chat URL:
```

Comment:
This could also be a GitHub discussion link I guess? Since there is no way to convert a GitHub discussion to a GitHub issue.